tests/Database/DatabaseEloquentFactoryTest.php

```diff
@@ -838,12 +839,15 @@ public function test_factory_model_names_correct()
 
     public function test_factory_global_model_resolver()
     {
-        \Illuminate\Database\Eloquent\Factories\Factory::guessModelNamesUsing(function ($model) {
```

Comment:
`$model` here is an instance of `Illuminate\Database\Eloquent\Factories\Factory` and not a string as indicated in the test (return). If the code was executed this would have thrown an error instead of passing the test.